### PR TITLE
Create feature-file2.txt

### PR DESCRIPTION
**Pull request by jahangir1x**
some new feature
feature
fdkkfd kfkdf

### CODE REVIEW

**Comment by zinis** [comment]
fix xyz

File: feature-file2.txt [hunk 1]
```diff
@@ -0,0 +1,3 @@
+some line 1
+some line 2
+adsaf asdf
```
Comment (zinis) [inline feature-file2.txt:3]: what is this?

Comment (zinis) [reply feature-file2.txt:3]: hey

**Comment by jahangir1x** [comment]
yep

File: feature-file2.txt [hunk 2]
```diff
@@ -0,0 +1,3 @@
+some line 1
```
Comment (zinis) [inline feature-file2.txt:1]: hey what is this?

File: feature-file2.txt (hunk 1 above)
Comment (jahangir1x) [reply feature-file2.txt:3]: beep boop

File: feature-file2.txt (hunk 2 above)
Comment (jahangir1x) [reply feature-file2.txt:1]: bada boom

File: feature-file2.txt [hunk 3]
```diff
@@ -0,0 +1,3 @@
...
+some line 2
```
Comment (zinis) [inline feature-file2.txt:2]: yeeee?

**Review by zinis — CHANGES_REQUESTED** [review]

**Comment by zinis** [comment]
yeeee.

**Pull request merged**